sympy/polys/matrices/domainmatrix.py

```diff
@@ -234,3 +244,68 @@ def __eq__(A, B):
         if not isinstance(B, DomainMatrix):
             return NotImplemented
         return A.rep == B.rep
+
+
+def __getindex__(i, end):
```

Comment:
Giving random functions a double underscore name is confusing. This is not a special method so it shouldn't have a dunder name.

Reply:
Sure.
Thanks.